recipe/recipe.go

```diff
@@ -159,6 +159,10 @@ func (r *Recipe) Parse(file string, templateVars ...map[string]string) error {
 		return err
 	}
 
+	if (dump) {
+		fmt.Println(data)
```

Comment:
Nitpick; would be nice to print a header when doing this.

Reply:
Done